hvplot/__init__.py

```diff
@@ -127,6 +130,13 @@ def _patch_doc(cls, kind):
         method.__signature__ = signature
 
 
+@_wraps(_pn.bind)
```

Comment:
I'm not sure Panel's docstring for `bind` will be very understandable by hvPlot users.